docs/integrations/skopt.rst

```diff
@@ -51,101 +106,90 @@ Everything can be inspected live.
     results = skopt.forest_minimize(objective, space, callback=[neptune_callback],
                                     base_estimator='ET', n_calls=100, n_random_starts=10)
 
-Log all results
----------------
-You can log additional information from skopt results after the sweep has completed.
-By running:
+Step 5: Monitor your Skopt tuning in Neptune
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now you can switch to the Neptune tab which you had opened previously to watch the tuning live!
 
-.. code-block:: python3
+|Run on Colab|
 
-    skopt_utils.log_results(results)
+.. _skopt-advanced-options:
 
-You log the following things to Neptune:
+More Options
+------------
 
-* Best score
-* Best parameters
-* Figures from plots module: plot_evaluations, plot_convergence, plot_objective, and plot_regret
-* Pickled results object
+You can log additional information from skopt results after the tuning has completed.
 
 .. code-block:: python3
 
     skopt_utils.log_results(results)
 
-Monitor your Scikit-Optimize training in Neptune
-------------------------------------------------
-Now you can watch your Scikit-Optimize hyperparameter optimization in Neptune!
+You can change the Neptune experiment to which the results are logged with the ``experiment`` parameter, and choose whether or not you want to log plots and the pickle objects with the ``log_plots`` and ``log_pickle`` parameters. 
 
-Check out this |example experiment|.
+More information about the ``log_results()`` method |log_results|.
```

Comment:
In rst you can actually link to the method by doing:
```
:meth:`~neptune.experiments.Experiment.log_metric`
```
So in this case it would be
```
:meth:`~neptunecontrib.monitoring.skopt.log_results`
```